tests/suites/0_stateless/02_function/02_0000_function_aggregate_mix.sql

```diff
@@ -23,4 +23,41 @@ SELECT std(number) between  2886.751 and 2886.752 from numbers_mt(10000);
 SELECT stddev(number) between  2886.751 and 2886.752 from numbers_mt(10000);
 SELECT stddev_pop(number) between  2886.751 and 2886.752 from numbers_mt(10000);
 SELECT covar_samp(number, number) from (select * from numbers_mt(5) order by number asc);
-SELECT covar_pop(number, number) from (select * from numbers_mt(5) order by number asc);
\ No newline at end of file
+SELECT covar_pop(number, number) from (select * from numbers_mt(5) order by number asc);
+
+DROP DATABASE IF EXISTS db1;
+CREATE DATABASE db1;
+USE db1;
+
+SELECT '==Variant==';
+CREATE TABLE IF NOT EXISTS t1(id Int null, var Variant null) Engine = Memory;
```

Comment:
I prefer to remove the `Engine = Memory`. Let's use `fuse` engine by default for all the tests.

Reply:
ok